src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Data/Json/GetRules/GetRules_RuleTypeInheritedAsSubunitRequest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parentPartyId": 50001337,
```

Comment:
OfferedByPartyId and ParentPartyId set to the same PartyId in the request shouldn't be able to give a correct InheritedAsSubunit scenario?

Reply:
Fixed SetRuleType so it sets the correct ruletype, and updated the unit tests